articles/rules/current/index.md

```diff
@@ -349,7 +349,7 @@ Notice that the code sandbox in which Rules run on, can be recycled at any time.
 
 ## Available modules
 
-For security reasons, the Rules code runs in a JavaScript sandbox based on [webtask.io](https://webtask.io) where you can use the full power of the ECMAScript 5 language.
+For security reasons, the Rules code runs in a JavaScript sandbox based on [webtask.io](https://webtask.io) where you can use the full power of the ECMAScript 6 language.
 
 For a list of currently supported sandbox modules, see [Modules Supported by the Sandbox](https://tehsis.github.io/webtaskio-canirequire) and [Additional Modules Available in Rules](/appliance/modules).
```

Comment:
FYI The "canirequire" mechanism is going to undergo changes soon. For context see: https://auth0.slack.com/archives/C9170558S/p1529349217000479

Reply:
Thanks, I followed that issue so when the change is ready I can make any updates